src/style.css

```diff
@@ -0,0 +1,98 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  flex-basis: 100%;
+
+  height: 300px;
+  min-width: 300px;
+
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.block1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block6 {
+  background-color: rgb(0, 0, 0);
+}
+
+@media screen and (min-width: 600px) and (max-width: 900px) {
+  .block {
+    flex-basis: 50%;
+  }
+
+  .block1,
+  .block2 {
+    order: -1;
+  }
```

Comment:
If you provide custom order, don't give same values to several elements. Use unique numbers.